### PR TITLE
configure input capitalization

### PR DESCRIPTION
**Pull request by yzhang123**
* adding configuration to distinguish between lower_cased and cased input, affects whitelist, word classes
* added test
* added pynini sentence predict test to jenkins
* deleted sentence boundary exception

Signed-off-by: Yang Zhang <yangzhang@nvidia.com>

### CODE REVIEW

File: nemo_text_processing/text_normalization/data/sentence_boundary_exceptions.txt [hunk 1]
```diff
@@ -1,15 +1,4 @@
-mr.
```
Comment (ekmb) [inline nemo_text_processing/text_normalization/data/sentence_boundary_exceptions.txt:1]: Do we need this file or can we use the whitelist? added to our to-do

File: nemo_text_processing/text_normalization/normalize.py [hunk 2]
```diff
@@ -22,10 +22,11 @@
 from nemo_text_processing.text_normalization.verbalizers.verbalize_final import VerbalizeFinalFst
 from tqdm import tqdm
 
+INPUT_CASE = "lower_cased"  # cased
```
Comment (ekmb) [inline nemo_text_processing/text_normalization/normalize.py:25]: can we infer this from the input itself?  and in the case when it's not possible, use this param? could you add it to the arguments?

**Review by ekmb — CHANGES_REQUESTED** [review]
Please add test cases to cover the changes.

File: nemo_text_processing/text_normalization/data/sentence_boundary_exceptions.txt (hunk 1 above)
Comment (yzhang123) [reply nemo_text_processing/text_normalization/data/sentence_boundary_exceptions.txt:1]: thanks, yes its todo

File: nemo_text_processing/text_normalization/normalize.py (hunk 2 above)
Comment (yzhang123) [reply nemo_text_processing/text_normalization/normalize.py:25]: let me figure out how to add it to args

**Pull request merged**